lib/pbench/cli/server/shell.py

```diff
@@ -62,24 +68,40 @@ def main():
             logger.info("Created DB {}", db)
         Database.init_db(server_config, logger)
     except (NoOptionError, NoSectionError):
-        logger.exception(f"{__name__}: ERROR")
+        logger.exception(f"{PROG}: ERROR")
```

Comment:
Not strictly needed as the logger created by `get_pbench_logger()` will already format in the context (see [lib/pbench/common/logger.py#L169](https://github.com/distributed-system-analysis/pbench/blob/main/lib/pbench/common/logger.py#L169), `name` refers to the name of the logger used [LogRecord Attributes](https://docs.python.org/3/library/logging.html#logrecord-attributes)).

You didn't write this originally, but a separate PR could consider changing this to:

```
logger.exception("Error fetching required configuration setttings")
```

Reply:
FWIW:

```
$ cat this.py
def main():
    print(__name__)

if __name__ == "__main__":
    main()
$ cat that.py
import this

this.main()
$ python3 this.py
__main__
$ python3 that.py
this
$
```

It is likely that the `gunicorn` logger was being used and as such the settings on the logger were working against what we want for Pbench.  So this is a good change to make.

Reply:
Followed-up by PR #3042.